<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>550</y>
     <width>621</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter your DNA sequence:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Get multipeptide:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Get RNA:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>341</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>270</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_4">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>370</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>230</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pourcentage of C and G</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>320</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Pourcentage of A and U</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>250</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>in RNA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>350</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>in RNA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Your DNA sequence:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>341</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Your RNA sequence:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>390</y>
     <width>341</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Your multipeptide sequence is:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 20pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>DNA, RNA, Protien Translator </string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../dna2rna2prt/source.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
